File: leetcode/1easy/오하늘/1961.py
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution:
=======
class Leetcode.이재훈.Solution:
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    def isPrefixString(self, s: str, words: List[str]) -> bool:
        ans = ""
        for word in words:
            ans += word
            if ans == s:
                return True
            if len(ans) > len(s):
                return False
        return False

File: leetcode/1easy/이현진/1961.js
const isPrefixString = function (s, words) {
  let concatenated = "";

  for (let word of words) {
    concatenated += word;
    if (concatenated === s) {
      return true;
    }
    if (concatenated.length > s.length) {
      return false;
    }
  }

  return false;
};
